frontend/src/components/RecordsList.tsx

```diff
@@ -91,8 +91,14 @@ export default function RecordsList({
 
   const dataNameCapitalized = buildSentenceCaseString(dataName);
   const recordsQueryName = `dashboard${dataNameCapitalized}`;
-  const recordCount = data?.[recordsQueryName][0]._total;
-  const uniqueSampleCount = data?.[recordsQueryName][0]._uniqueSampleCount;
```

Comment:
Nice. JS has built-in operators specifically for this scenario. Could we use those instead? For example:
```ts
  const recordCount = data?.[recordsQueryName][0]?._total || 0;
  const uniqueSampleCount = data?.[recordsQueryName][0]?._uniqueSampleCount || 0;
```